addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -127,6 +127,20 @@ var ActivityMenu = Widget.extend({
         }
     },
 
+    /**
+     * Return views to display when coming from systray depending on the model.
+     * @private
+     */
+    getViewsList: function(model){
+        var views = [[false, 'kanban'], [false, 'list'], [false, 'form']];
+        switch(model){
+            case 'crm.lead':
+                views = views.concat([[false, 'calendar'], [false, 'pivot'], [false, 'graph'], [false, 'activity']]);
+                break;
```

Comment:
This is breaking the module structure: `crm` is not a dependency of `mail` so it shouldn't be referenced here.

This method should only have the default value, and be extended appropriately by other modules.

Ideally, this shouldn't even have to be defined, unless we want specific cases, as the relation between models and the available views are already known by the system since they are defined in data files.

Reply:
The PO (Luc Nailis) don't want to display all the available views for each model when coming from the systray then it need to be defined. But yes it make more sense to extend this method in the appropriate module to add the model specific's views.